create-projects/create-projects.sh

```diff
@@ -80,6 +80,17 @@ if [ -n "${PROJECT_ADMINS}" ]; then
     oc policy add-role-to-user admin "${admin_user}" -n "${PROJECT_ID}-test"
     oc policy add-role-to-user admin "${admin_user}" -n "${PROJECT_ID}-cd"
   done
+  
+  echo "Assign the owner as a label to the OpenShift project"
+  # Take the first in the list as owner of the project.
+  # Labels do not allow the '@' char so let's replace it with '_at_'
+  # For instance, foo@bar.com will be converted to foo_at_bar.com
+  # resulting in the following label: ods.project.owner=foot_at_bar.com
+  namespace_owner=$(echo "${PROJECT_ADMINS}" | cut -d "," -f1 | sed -r 's/@/_at_/g')
```

Comment:
I'm assuming `PROJECT_ADMINS` is a comma-separated list of email addresses, could anyone confirm?

Reply:
There are tests for this :) https://github.com/opendevstack/ods-core/blob/master/create-projects/tests/run.sh